dss/util/dynamodb.py

```diff
@@ -0,0 +1,65 @@
+from dss.util.aws.clients import dynamodb  # type: ignore
+
+
+class DynamoLookup(object):
```

Comment:
A class with nothing but static methods may not deserve to be a class.

Note that modules can provides descriptive name spacing.
```
imporot my_cool_module

my_cool_module.my_cool_method()
```

Reply:
It's not just for descriptive namespacing.  I prefer an ABC to importing bare functions because it clearly defines which methods are intended to be inherited and some methods will only need to be wrapped and returned.

I'd be willing to make this purely functional if you prefer, I think it's more of a style preference.  I will argue that it makes more sense alongside the collections class also inheriting the same methods, at least to me it looks nicer.